routers/api/packages/api.go

```diff
@@ -118,6 +118,11 @@ func CommonRoutes(ctx gocontext.Context) *web.Route {
 					r.Get("/owners", cargo.ListOwners)
 				})
 			})
+			r.Get("/config.json", cargo.RepositoryConfig)
+			r.Get("/1/{package}", cargo.EnumeratePackageVersions)
+			r.Get("/2/{package}", cargo.EnumeratePackageVersions)
+			r.Get("/3/{_}/{package}", cargo.EnumeratePackageVersions)
+			r.Get("/{_}/{__}/{package}", cargo.EnumeratePackageVersions)
```

Comment:
What's the `_` and `__` part? I guess it's omitted explicitly, could there be some comments?

Reply:
Yes, just dummy placeholders. I will add a comment. Cargo uses `/he/ll/hello_world` as path for a package named `hello_world`. We could enforce a correct path so that `/ab/cd/hello_world` does not work but I don't see a real benefit.